casper/src/test/scala/coop/rchain/casper/MergeabilityRules.scala

```diff
@@ -446,19 +544,98 @@ trait MergeabilityRules {
     "(C!) (C!)" -> PersistentNoMatch(C_, S0)(C_, S0)(Nil)
   )
 
+  val peekMergeabilityCases = List(
+    "!4 P!"     -> HadItsMatch(S0)(P_)(F0, S1),
+    "!4 (P!)"   -> PeekedNoMatch(S0)(P_, S1)(F0),
+    "!X PX"     -> IncomingCouldMatch(S0)(P_)(Nil),
+    "!4 PX"     -> HadItsMatch(S0)(P_)(F_),
+    "!C PX"     -> HadItsMatch(S0)(P_)(C_),
+    "!X (P!)"   -> PeekedNoMatch(S0)(P_, S0)(Nil),
+    "!X P!"     -> HadItsMatch(S0)(P_)(S0),
+    "(!4) P!"   -> VolatileEvent(S0, F0)(P_)(S1),
+    "(!4) PX"   -> VolatileEvent(S0, F0)(P_)(Nil),
```

Comment:
I think it'd be helpful to prefix all mergability case names with `Merge / `Conflict`, e.g.:

```
MergeHadItsMatch
MergePeekedNoMatch
ConflictIncomingCouldMatch
MergeVolatileEvent
MergeSamePolarityMerge
MergeCouldMatchSame[Didnt]
ConflictCouldMatchSame[Did]
```